table/block_based/block_based_table_reader.cc

```diff
@@ -2187,6 +2232,60 @@ Status BlockBasedTable::MaybeReadBlockAndLoadToCache(
       }
     }
   }
+
+  if (block_cache_tracer_ && lookup_context) {
+    // Fill lookup_context.
```

Comment:
Should this line of comment be moved closer to the actual filling of `lookup_context`?

Reply:
Moved up before the if statement.